doc/dev/crimson/crimson.rst

```diff
@@ -64,6 +65,57 @@ The last causes pools to be created by default with the ``crimson`` flag.
 Crimson pools are restricted to operations supported by Crimson.
 ``Crimson-osd`` won't instantiate PGs from non-Crimson pools.
 
+Configure Crimson with Bluestore
+================================
+
+Deploying Crimson with Bluestore as the back end requires setting
+several configuration values. The required options are
+``crimson_seastar_cpu_cores`` and ``crimson_alien_thread_cpu_cores``.
+
+Bluestore is not a Crimson native `object store backend`_.
+Therefore, we must explictily set the CPU core allocation for each ``crimson-osd``
+and for the BlueStore back end. It's recommended for each set to be mutually exclusive.
```

Comment:
How about:

Because crimson requires threads to be pinned to distinct cores, the user must specify for each osd which cores should be used for crimson seastar reactor threads (``crimson_seastar_cpu_cores``) and for the bluestore thread pool threads ``crimson_alien_thread_cpu_cores``.

Reply:
I have added (a slightly edited version of) this sentence to this file.